cogs/subscribers.py

```diff
@@ -122,11 +135,11 @@ def check_status(line_number, response):
             response = requests.request("GET", METRO_STATUS_API)
             for line_status in metro_status.items():
                 line_number = line_status[0]
-                cached_status = line_status[1][0]
-                line_colour = line_status[1][1]
+                cached_status = line_status[1]
+                line_colour = METRO_COLOURS[line_number]
```

Comment:
this variable is probably unnecessary since it's just used once

Reply:
Resolved by d13c353.